util/prep-bazel-airgapped-build.sh

```diff
@@ -156,8 +157,17 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
     @rust_windows_x86_64_toolchains//...
   cp -R "$(${BAZELISK} info output_base)"/external/${BAZEL_PYTHON_WHEEL_REPO} \
     ${BAZEL_AIRGAPPED_DIR}/
-  cp -R "$(dirname "$(readlink -f "$(${BAZELISK} info output_base)"/external/${BAZEL_BITSTREAMS_REPO}/cache)")" \
-    ${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}
+  # We don't need all bitstreams in the cache, we just need the latest one so
+  # that the cache is "initialized" and "offline" mode will work correctly.
+  mkdir -p ${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}
+  readonly SYSTEM_BITSTREAM_CACHE="${HOME}/.cache/opentitan-bitstreams"
+  readonly SYSTEM_BITSTREAM_CACHEDIR="${SYSTEM_BITSTREAM_CACHE}/cache"
```

Comment:
Out of curiosity, was there a motivation for choosing this hard-coded mechanism for grabbing the source bitstream cache directory? Previously, it had used repository set up in bazel.

Reply:
The previous setup was not quite right and I was surprised it worked actually. The previous setup copied the Bazel external repo directory that got generated by the `bitstreams_workspace.py` script into the airgapped directory. (This also copied all the bitstreams that were symlinked within this folder, when only one is really needed.) What was really desired was to copy the custom bitstreams "cache" folder on the system to the airgapped dir so that when bazel runs in airgapped mode the `bitstreams_workspace.py` script still runs normally (this time in the "--offline" mode) and the Bazel external repo directory gets generated on the airgapped machine as expected. This enables a cleaner mechanism to retrieve the single (latest) bitstream to copy over (I just used the hash in the `latest.txt` file) as opposed to having to parse the filegroup sources defined the Bazel.build file generated by  the `bitstreams_workspace.py` script, and hoping these name formats never change.

On the flip side, this resulted in hardcoding the default path to the system cache used by the Bazel [rules](https://cs.opensource.google/opentitan/opentitan/+/master:rules/bitstreams.bzl;drc=8365ac4e97ca340e441ff3355fa07862449b9c8d;l=61).

Admittedly, these will have to stay in sync, so perhaps there is a more maintainable solution out there?

But all in all, I didn't want to just  blindly copy the Bazel external repo directory that gets generated by the `bitstreams_workspace.py` script, only to let it get (potentially) modified again by Bazel invoking the script again on the airgapped machine using the [`--cache` parameter](https://cs.opensource.google/opentitan/opentitan/+/master:rules/bitstreams.bzl;drc=8365ac4e97ca340e441ff3355fa07862449b9c8d;l=19). https://cs.opensource.google/opentitan/opentitan/+/master:rules/bitstreams.bzl;drc=8365ac4e97ca340e441ff3355fa07862449b9c8d;l=19 since